tests-perf/locust/locust-notifications.py

```diff
@@ -12,13 +12,50 @@ class NotifyApiUser(HttpUser):
     wait_time = constant_pacing(60)
     host = "https://api.staging.notification.cdssandbox.xyz"
 
-    @task
-    def send_notifications(self):
-        headers = {"Authorization": AUTH_HEADER}
-        json = {
+    def __init__(self, *args, **kwargs):
+        super(NotifyApiUser, self).__init__(*args, **kwargs)
+
+        self.headers = {"Authorization": AUTH_HEADER}
+        self.json = {
             "email_address": "success@simulator.amazonses.com",
             "template_id": "9c17633c-126a-4ad3-ad2f-b14c3a85314a",
             "personalisation": {"colour": "Fulvous"},
         }
 
-        self.client.post("/v2/notifications/email", json=json, headers=headers)
+    @task
```

Comment:
If we have the same configuration for all user classes, we can have all `task` annotations in the same class and put different weight in each to determine which one would get used, [per the documentation.](https://docs.locust.io/en/stable/writing-a-locustfile.html#task-decorator)